packages/astro/src/core/sync/index.ts

```diff
@@ -116,8 +118,9 @@ export async function syncInternal({
 	force,
 	manifest,
 }: SyncOptions): Promise<void> {
+	const isDev = mode === 'development';
```

Comment:
Double checking if this is correct. In Astro, you can also technically to `astro build --devOutput` to get the development build. Do we want `isDev` `true` in that case?

The `mode` here is typed as `string` because end-users can name it anything in practice. e.g. `astro build --mode staging` or `astro build --mode staging --devOutput`.

Maybe passing the `runtimeMode` from `build()` would be more accurate for `isDev`.

Reply:
What we're actually wanting to know is if they're running `astro dev`. It doesn't matter what the build output is: it's the only thing we care about is whether we need the chokidar watcher to see the data store.

Reply:
I think what you committed is good then if that's the case 👍